api_app/analyzers_manager/migrations/0079_analyzer_config_permhash.py

```diff
@@ -0,0 +1,128 @@
+from django.db import migrations
```

Comment:
please also add this analyzer to the freely available playbook + to the static analyzer playbook